services/backups/service.py

```
@@ -642,21 +696,47 @@ def preview_delete(self, capture_ids: Optional[List[str]] = None,
         selected, skipped_pinned = self._select_for_delete(
             capture_ids, slot_keys, keep_newest, include_pinned,
         )
+
+        captures = []
+        for index, record in enumerate(selected):
```

Ruff (0.16.1)

[error] Import `record` from line 33 shadowed by loop variable

(F402)